recipe/migrations/google_cloud_cpp250.yaml

```diff
@@ -1,14 +1,14 @@
 __migrator:
   build_number: 1
   kind: version
-  migration_number: 1
+  migration_number: 2
```

Comment:
There wouldn't have been a need to bump this number; there's only one consumer for google-cloud-cpp (it's arrow), and now that feedstock got spammed with a second round of 5 PRs for all the maintenance branches (additionally because there are still no users of `libgoogle-cloud-cpp` that I'm aware of, the typo was irrelevant), after I already merged the first round... 😑

Reply:
If you want to conserve some CI resources, you can cancel the builds for PRs 906-910

Reply:
hmm. yeah. i didn't realize it was arrow. I fiured it was "harmless" to do it so quickly.

Reply:
oh. i don't know how to cancel builds either.